util/manage-page-order.p6

```diff
@@ -48,20 +53,89 @@ given $mode {
     }
 }
 
-sub do-update() {
+sub do-update () {
     say "DEBUG: in sub {&?ROUTINE.name}" if $debug eq 'r';
+
+    # FIRST
+    # write the auto-generated pod6 files used for the Language page
+    write-Language-files();
+
+    # THEN
+    # copy all under the doc dir to the build dir
+    # NOTE this assumes the sort order is alpha by file name
+    # NOTE we exclude the Language dir
+    copy-dir-tree(:fromdir('doc'), :todir('build'), :exclude('Language'));
+}
+
+=begin comment
```

Comment:
Commented out code?

Reply:
I was savng it until all checked out—it can be removed.